homeassistant/components/smartthings/strings.json

```
@@ -604,6 +604,9 @@
       "bubble_soak": {
         "name": "Bubble Soak"
       },
+      "air_conditioner_lighting": {
+        "name": "Air conditioner lighting"
```

As I understand it, this setting only appears for air conditioner device entries?

Then it makes no sense to repeat "Air conditioner" in the name of the switch.
Instead "Display" needs to be added so it is clear that this is not to illuminate the whole room:

```suggestion
        "name": "Display lighting"
```

You might want to adapt the translation key, too.